richtextfx/src/main/java/org/fxmisc/richtext/TextExt.java

```diff
@@ -291,6 +333,16 @@ private static class StyleableProperties {
                 0, n -> n.underlineWidth
         );
 
+        private static final CssMetaData<TextExt, Number> UNDERLINE_OFFSET = new CustomCssMetaData<>(
+            "-rtfx-underline-offset", StyleConverter.getSizeConverter(),
+            0, n -> n.underlineOffset
+        );
+
+        private static final CssMetaData<TextExt, Number> UNDERLINE_WAVE = new CustomCssMetaData<>(
+            "-rtfx-underline-wave", StyleConverter.getSizeConverter(),
```

Comment:
Let's make this UNDERLINE_WAVE_RADIUS and "-rtfx-underline-wave-radius" please. I know it's more verbose but I think it adds clarity, since we've already got "-rtfx-underline-dash-array".